### ISSUE TITLE
Implement functionality to enter power management use case area 

### ISSUE THREAD

**Issue by stefanieschreiner** [i315315/amos2022ws02-automotive-test-app#25]
## User story
1. As a User
2. I want to be able to enter an area in the test app where all use cases for simulating power management are collected 
3. So that I can easily switch between the use cases.4. 

With this User story the functionality of entering the area for power management use cases is to be implemented.

## Acceptance criteria
* Criterion 1
* Criterion 2
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
